bundles/org.openhab.binding.alarmdecoder/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="alarmdecoder"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Alarm Decoder Bridge -->
+	<bridge-type id="ipbridge">
+		<label>Alarm Decoder IP Bridge</label>
+		<description>Nu Tech Alarm Decoder IP Bridge</description>
+		<properties>
+			<property name="vendor">Nu Tech Software Solutions</property>
+		</properties>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<label>Host Name</label>
+				<context>network-address</context>
+				<description>The hostname or IP address of the Alarm Decoder device</description>
+			</parameter>
+			<parameter name="tcpPort" type="integer">
+				<label>TCP Port</label>
+				<description>TCP port number for the Alarm Decoder connection</description>
+				<default>10000</default>
+			</parameter>
+			<parameter name="discovery" type="boolean">
+				<label>Enable Discovery</label>
+				<description>Enable automatic discovery of zones and RF zones</description>
+				<default>false</default>
+			</parameter>
+			<parameter name="reconnect" type="integer" min="1" max="60" unit="min">
+				<label>Reconnect Interval</label>
+				<description>The period in minutes that the handler will wait between connection attempts and checks</description>
+				<unitLabel>minutes</unitLabel>
+				<default>2</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="timeout" type="integer" min="0" max="60" unit="min">
+				<label>Message Receive Timeout</label>
+				<description>The period in minutes after which the connection will be reset if no valid messages have been received.
+					Set to 0 to disable.</description>
+				<unitLabel>minutes</unitLabel>
+				<default>5</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<bridge-type id="serialbridge">
+		<label>Alarm Decoder Serial Bridge</label>
+		<description>Nu Tech Alarm Decoder Serial Bridge</description>
+		<properties>
+			<property name="vendor">Nu Tech Software Solutions</property>
+		</properties>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<label>Serial or USB Port</label>
+				<context>serial-port</context>
+				<description>The name of the serial port used to connect to the Alarm Decoder device</description>
```

Comment:
```suggestion
				<description>The name of the serial port used to connect to the Alarm Decoder device.</description>
```

Reply:
Well, this isn't actually a sentence, so I wouldn't normally put a period there. The parallel I would draw is sentence fragments in a bullet list. It's debatable, though. Do these description fields usually have periods?